packages/react-ui-core/src/DateInput/DateInput.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
```

Comment:
Isn't this just an empty wrapper that passes props through to the `DatePicker`?
Would something closer to what you wrote for the story's component maybe be better? idk, just throwing that out there

Reply:
Its just wrapper to pass props to DatePicker. In story book, we have default behaviour for Datepicker.